.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: "🐞 Bug report"
+about: Create a report to help us improve
+labels: kind/bug
+---
+
+> Bugs should be filed for issues encountered whilst operating the Tekton Dashboard.
+> You should first attempt to resolve your issues through the community support channels,
+> e.g. `#dashboard` channel in the [Community Slack](https://github.com/tektoncd/community/blob/master/contact.md#slack).
+> Please provide as much detail as possible.
+
+# Describe the bug
+
+A clear and concise description of what the bug is.
+
+# Expected behaviour
+
+A concise description of what you expected to happen.
+
+# Steps to reproduce the bug
+
+Steps to reproduce the bug should be clear and easily reproducible to help people gain an understanding of the problem.
+
+# Environment details
+
+- Kubernetes Platform:
+<!-- (e.g. Docker Desktop, OpenShift) -->
+- Kubernetes or OpenShift version:
+<!-- (e.g. v1.15.2) -->
+- Install mode (if on OpenShift):
+<!-- (e.g. operator, non-operator) -->
+- Cloud-provider/provisioner:
+<!-- (e.g. AKS, GKE, EKS, PKE) -->
+- Tekton Dashboard version:
+<!-- (e.g. 0.3.0, 0.4.0, master) -->
+- Tekton Pipelines version:
+<!-- (e.g. 0.8.0, 0.9.2, master) -->
+- Tekton Triggers version:
+<!-- (e.g. 0.1, 0.2, master) -->
```

Comment:
Could we do something like this to match the namespaces below?

- Versions:
  - Tekton Dashboard:
  - Tekton Pipelines:
  - Tekton Triggers:

Reply:
fixed